cassandra-bridge/src/main/java/org/apache/cassandra/spark/data/CqlField.java

```diff
@@ -263,7 +263,7 @@ public CqlField(boolean isPartitionKey,
         this.isPartitionKey = isPartitionKey;
         this.isClusteringColumn = isClusteringColumn;
         this.isStaticColumn = isStaticColumn;
-        this.name = name.replaceAll("\"", "");
+        this.name = name;
```

Comment:
we no longer need to drop the quotes, since we no longer pass the quoted string from here: https://github.com/apache/cassandra-analytics/pull/19/files#diff-143d0bc336629bfe78bf9f0c64d83803701d38b22a4be736323277862b784b25R522

Reply:
Handling of UDTs is different from handling of keyspace / table. For UDTs, the bridge was already taking care of preserving the quotes. However, for user facing options (the keyspace and table specifically), we need special handling based on whether the user explicitly requests to quote identifiers

Reply:
ok, took a deeper look. I think the confusion is mainly on the EndToEnd testing code. So what I've done is address the test setup in testing code. I have made it so that we always quote the fields, when quotes are needed for testing purposes. 

I have also added a test case that validates the column filter (`PartialRowBuilder` class). That way we guarantee that we are able to retrieve columns that are requested in the schema where columns are case sensitive in Cassandra